src/main/view/GameModelAdapter.java

```diff
@@ -0,0 +1,18 @@
+package view;
```

Comment:
what's the purpose of the adapter? Is this going to be what handles updating of simple tiles based on visibility?